### PR TITLE
Web App adjustments from issue #375 #387

### PR DESCRIPTION
**Pull request by ruoyuch**
Trying to provide a clearer Web App.  I added _new tooltips_ (still keeping it short) and _explanations_ (on the running time).  Also, I try to add a _download report button_.  However, the local file is not as easy to access as I expected.  I commented on the codes for the button; if you have any ideas or references on the local file download code, please let me know.

### CODE REVIEW

**Comment by carlhiggs** [comment]
Hi @rychennn, thanks for your work on this --- sounds like some good updates here, but I notice that there are conflicts with the parent branch from other updates that have occurred.  Could you merge the updates from the parent branch into your version and resolve the conflicts and then lodge the pull request again?

Regarding a button to download the report, this is an interesting idea; two challenges I see:
- filenames (validation report and analysis report) currently include a date or year to indicate when generated, so this is not something that is static.  For this reason the actual filename is currently determined not in the r.config data per se, but in the function that generates the file (e.g. [here](https://github.com/global-healthy-liveable-cities/global-indicators/blob/fbb9697cd81a63c4f5b270aedf715e905a273a46/process/subprocesses/_utils.py#L1361))
- there are multiple reports (validation reports, and various kinds of indicator reports potentially in various languages)

So, I think the solution may be not to have a single button, but perhaps identify all the files that are either '.pdf' in the selected study region's output folder, or otherwise have partial names that indicate whether they are validation or analysis reports.   Buttons/links to view/download PDF files could then be displayed for all the identified resources.   

So, in short, maybe don't try to predict or retrieve the exact filename.  Instead, identify all/specific types of pdf for selected study region and then display the download links for these.

Hope this is useful,

Carl

**Comment by ruoyuch** [comment]
Hi Carl,

Sorry for the delayed response. I probably missed the notification from GitHub for some reason. I will merge the updates and solve the conflicts very soon—I will open a new pull request after solving the conflicts locally, which is a bit messy to solve in the web browser.

Regarding the download button, I think the solutions you proposed sound very promising; one problem I faced before was picking up local files when operating the NiceGUI. According to my experience, NiceGUI cannot use a relative path to get the local files due to security reasons, and I tried the local_file_picker (which was used by you as well) but failed.  I will spend some more time on this in the coming weeks to see whether I can make some helpful changes.

Thanks,
Ruoyu

**Comment by carlhiggs** [comment]
No worries, and sounds good @rychenn. Thanks for looking into this!
